src/main/java/elucent/rootsclassic/block/altar/AltarBlockEntity.java

```diff
@@ -80,19 +102,31 @@ public void load(CompoundTag tag) {
 	public void saveAdditional(CompoundTag tag) {
 		super.saveAdditional(tag);
 		tag.put("InventoryHandler", inventory.serializeNBT());
-		if (getIncenses().size() > 0) {
```

Comment:
Hey, I get that a lot of the changes you have made are likely nice ones but just a tip for the future, try to keep your PR's as close to only the changes needed rather than nice to have changes, especially if they are of this scale. It makes the pr a lot harder to review and more likely to have clashes.

Reply:
I am sorry, I realize now that I have probably made a lot of these unnecessary changes automatically, without thinking about it